queries/ecma/highlights.scm

```diff
@@ -138,6 +140,17 @@
   (call_expression
     (identifier) @attribute))
 
+(decorator
+  "@" @attribute
+  (member_expression
+    (property_identifier) @attribute))
```

Comment:
Can there be `@A.B.C`? Because if yes, I think we might be in a sticky situation.

Reply:
I think it's fine:
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/4299733/da5652b8-bbc6-4219-8008-1cf07f897cf9)

Reply:
I'm unsure if `@A.B.C` is semantically correct, but it's usually just `@SomeModule.Decorate()`.  
IMO from a syntactical perspective it's good enough.